pages/prospects/profile/[id].tsx

```diff
@@ -49,37 +41,44 @@ function ProspectProfile(props) {
     return (
       <Box sx={{ display: 'flex', justifyContent: 'center' }}>
         <Box sx={{ fontSize: '2rem', marginTop: '2rem' }}>
-            Page is Loading...
+          Page is Loading...
         </Box>
       </Box>
     )
-  }  
+  }
+
   if (hasError) {
     return (
-        <CustomError
-          title="User not found"
-          message="The user you are searching for does not exist."
-        />
+      <CustomError
+        title="User not found"
+        message="The user you are searching for does not exist."
+      />
     )
-  } 
+  }
+
   return (
-    <Paper sx={{ p: 1, margin: '2%' }}>
+    <Paper style={{ padding: 10, margin: '2%' }}>
       <Grid container spacing={2}>
         <Grid item xs={4}>
           <ProfileInfo prospect={prospect} />
         </Grid>
         <Grid item>
-          <Divider sx={{ height: '100%' }} orientation="vertical" />
+          <Divider style={{ height: '100%' }} orientation="vertical" />
         </Grid>
         <Grid item xs>
-          <Typography gutterBottom variant="h6">
+          <Typography gutterBottom variant="h6" style={{ flexGrow: 1 }}>
             Notes
           </Typography>
-          <Divider sx={{ width: 'auto' }} orientation="horizontal" />
+          <Divider style={{
+            width: 'auto',
+            marginBottom: 8
```

Comment:
why do you need this?

Reply:
I want to keep an exact distance between the divider and the 'textInput' that will open (I like the symmetry). 
I do the same for the 'Save' and 'Cancel' buttons.